c172p-set.xml

```diff
@@ -19,7 +19,7 @@ http://forum.flightgear.org/viewtopic.php?f=4&t=25157
 <PropertyList include="Aircraft/Generic/Human/Include/walker-include.xml">
 
     <sim include="c172p-views.xml">
-        <description>Cessna 172P Skyhawk (1981 model, detailed)</description>
+        <description>Cessna 172P Skyhawk (1982)</description>
 
         <long-description>The Cessna 172 Skyhawk is a four-seat, single-engine, high-wing fixed-wing aircraft. First flown in 1955 and still in production, more Cessna 172s have been built than any other aircraft.</long-description>
```

Comment:
You could write '172P' instead of '172' in the long descriptions?

Reply:
Just '172' is fine too.

Reply:
Maybe just leave it at '172' since it refers to the series.

Reply:
Yeah, adding a P would make the second phrase puzzling since the model P was not first flown in 1955.